hack/update-generated-defaulters.sh

```diff
@@ -25,8 +26,8 @@ EXTRA_PEER_DIRS=$(
 )
 
 gendefaults \
-	--logtostderr \
-	--output-base="${GOPATH}/src" \
-	--input-dirs ${INPUT_DIRS[@]} \
+  --logtostderr \
+  --output-base="${GOPATH}/src" \
+  --input-dirs ${INPUT_DIRS[@]} \
```

Comment:
These vars aren't arrays so I don't really understand the `[@]`. If you do make them arrays you need to quote this so it breaks on array elements and not spaces, if you don't make them arrays you need to quote this so it's always guaranteed to be one arg to `--input-dirs`